pkg/generate/template.go

```diff
@@ -153,6 +154,8 @@ type templateMetaVars struct {
 	// SDKAPIInterfaceTypeName is the name of the interface type used by the
 	// aws-sdk-go services/$SERVICE/api.go file
 	SDKAPIInterfaceTypeName string
+	//CRDNames contains all crds names lowercased and in plural
+	CRDNames []string
```

Comment:
This is cool for now. I think I'll clean up a lot of this template var stuff in a future PR to make things a bit easier/less repeated (notice that the `templateCmdVars` has a `SnakeCasedCRDNames` field that is very similar to what you're doing here with CRDNames :)

Reply:
@jaypipes cool thanks!